media.css

```diff
@@ -0,0 +1,457 @@
+  .wrapper {
+    width: 768px;
+    }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+    
+    /* margin-bottom: 33px;
+    padding-top: 24px; */
+    padding-bottom: 0;
+    margin-bottom: 26px;
+  }
+
+  .header__menu,
+  .header__search,
+  .header__user-block,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    width: 156px;
+    height: 34px;
+    display: flex;
+    align-items: center;
+    margin-left: 43px;
+  }
+  
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    margin-right: 44px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-768.png);
+    width: 100%;
+    height: 415px;
+    margin-bottom: 67px;
+  }
+
+ .top-content__block {
+    padding-left: 43px;
+    padding-top: 120px;
+  }
+
+  .top-content__h1 {
+    font-size: 50px;
+    line-height: 48px;
+    margin-bottom: 37px;
+  }
+
+  .link {
+    font-size: 20px;
+    line-height: 19px;
+  }
+  .hidden-768 {
+    display: none;
+  }
+  .catalog {
+    margin-left: 70px;
+    margin-right: 71px;
+  }
+  .catalog__title {
+    font-size: 30px;
+    line-height: 29px;
+    margin-bottom: 21px;
+  }
+  .catalog__item {
+    width: 281px;
+    height: 352px;
+  }
+  .catalog__item:not(:last-child):after {
+    content: "";
+    display: none;
+  }
+  .catalog-item__image {
+    width: 233px;
+    height: 230px;
+    margin-bottom: 20px;
+    padding-left: 28px;
+  }
+  .catalog-item__title {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 20px;
+    padding-left: 15px;
+  }
+  .catalog-item__description {
+    grid-template-columns: 1.5fr 1fr;
+    padding-left: 15px;
+    padding-right: 7px;
+  }
+  .catalog-item__price {
+    font-size: 25px;
+    line-height: 24px;
+  }
+  .catalog-item__grey,
+  .catalog-item__black,
+  .catalog-item__red,
+  .catalog-item__orange,
+  .catalog-item__brown,
+  .catalog-item__cherry,
+  .catalog-item__blue {
+    height: 20px;
+    width: 20px;
+  }
+  /*.video-section {
+    height: 490px;
+    background-color: #000000;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    /*padding-left: 30px;
+    margin-bottom: 117px;
+    /*position: relative;
+    overflow: hidden;
+    -webkit-transform: rotate(2.6deg);
+    transform: rotate(2.6deg);
+  }
+  .video-section:before {
+    content: "";
+    position: absolute;
+    width: 200%;
+    height: 200%;
+    top: -50%;
+    left: -50%;
+    z-index: -1;
+    background-color: #000000; 
+    -webkit-transform: rotate(-2.6deg);
+    transform: rotate(-2.6deg);
+    }
+  .video-section__image {
+    background-image: url(img/video.png);
+    display: block;
+    /*padding-left: 30px;
+  }
+
+  .video-section__content {
+    display: block;
+    /*width: 258px;*/
+    /*padding-top: 138px;*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)